packages/twoslash-vue/test/fixtures/cut-in-vue.vue

```diff
@@ -1,10 +1,9 @@
 <script setup lang="ts">
-// ---cut-start---
 import { ref } from "vue"
 interface Foo {
   bar: string
 }
-// ---cut-end---
+// ---cut---
```

Comment:
Instead of changing the fixture, can you create a new one so we can test with both cases?